src/bin/rustfmt.rs

```diff
@@ -7,23 +7,88 @@
 // <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
-
+#![feature(path_ext)]
 #![cfg(not(test))]
 
 extern crate rustfmt;
+extern crate toml;
 
 use rustfmt::{WriteMode, run};
+use rustfmt::config::Config;
+
+
+use std::env;
+use std::fs::{File, PathExt};
+use std::io::{self, Read};
+use std::path::{Path, PathBuf};
+
+use toml::Parser;
+
+// Tries to read a file and parse the conents as a toml table. Panics on error.
+fn read_toml<A: AsRef<Path>>(path: A) -> toml::Table {
```

Comment:
I think this function should also return a `Result` instead of panicing, since it might not be a fatal error.